### DISCUSSION TITLE
insufficient gas funds error on rinkeby infura using brownie

### DISCUSSION THREAD

**Discussion by moayaan1911** [smartcontractkit/full-blockchain-solidity-course-py#1501] [Q&A]
I have sufficient ether in my testnet account but still the code is showing error. What to do next? I have attached multiple screenshots of my code and metmask rinkeby wallet.. Kindly guide me

![Screenshot 2022-05-23 132627](https://user-images.githubusercontent.com/95223782/169773213-1ddcffa2-4451-42c6-b48f-8728b7cb8e28.png)
![Screenshot 2022-05-23 132709](https://user-images.githubusercontent.com/95223782/169773220-dd901b67-1214-4d22-8918-dfaf497f2a31.png)
![Screenshot 2022-05-23 132733](https://user-images.githubusercontent.com/95223782/169773230-1be9ee1d-e670-4a0c-9408-eb243f3aa648.png)
![Screenshot 2022-05-23 132745](https://user-images.githubusercontent.com/95223782/169773233-99b3a75a-f710-462e-827c-5617c9776d2e.png)
![Screenshot 2022-05-23 132800](https://user-images.githubusercontent.com/95223782/169773240-a39d87f8-7ddf-4f04-afb0-4716801f53e1.png)

**Comment by cromewar** [comment]
Hello @moayaan1911 
Please be sure you have added correctly the private key for your wallet,
remember to add a `.env` file and setup it on `brownie-config.yaml` .
And finally look up your `get_account` function to be sure you are returning the correct account.
